.buildkite/pipeline.yaml

```diff
@@ -1,14 +1,29 @@
 steps:
-  - label: "Build, lint, test"
+  - label: "Build, test, doc"
     commands:
-      - "ci/run"
-    agents:
+      - "ci/build-test"
+      - wait
```

Comment:
This is not the [wait step](https://buildkite.com/docs/pipelines/wait-step), but the bash builtin, which waits for child processes.

Reply:
:facepalm: